go/vt/topo/shard.go

```diff
@@ -578,3 +578,66 @@ func (ts *Server) GetTabletMapForShardByCell(ctx context.Context, keyspace, shar
 	}
 	return result, gerr
 }
+
+func (ts *Server) shardFilePath(keyspace, shard string) string {
```

Comment:
This can just be a package-level function since it doesn't need `ts`.

Reply:
Done